addons/portal/views/portal_templates.xml

```diff
@@ -428,7 +428,7 @@
             <div class="d-flex gap-2">
                 <img class="o_avatar o_portal_contact_img rounded" t-att-src="_contactAvatar" alt="Contact"/>
                 <div>
-                    <span class="d-block lh-1 mb-1" t-out="_contactName"/>
+                    <span class="d-block lh-1 mb-1 pt-1" t-out="_contactName"/>
```

Comment:
```suggestion
                    <span class="d-block lh-1 mb-1" t-att-class="{ 'pt-1': !_contactLink }" t-out="_contactName"/>
```

I think we should only vertically center the contact name if `Send Message` button is not displayed, otherwise the rendering could be a bit strange. 🤷🏼‍♂️ 

First screenshot before your changes and the second with your changes.

What do you think?

![image](https://github.com/user-attachments/assets/d255b1a9-197c-4af7-94fc-cd12989f5921)
![image](https://github.com/user-attachments/assets/95fbf645-5235-4425-b6dd-5a0ab75c5392)

Reply:
Yes, it does look better like that, I'll change it like you did